src/backend/distributed/commands/utility_hook.c

```diff
@@ -943,6 +955,8 @@ InvalidateForeignKeyGraphForDDL(void)
 {
 	if (shouldInvalidateForeignKeyGraph)
 	{
+		ereport(DEBUG1, (errmsg("DDL command invalidates foreign key graph")));
```

Comment:
I'd do this as DEBUG4 or so as this is a very low level information

Reply:
Makes sense but as I'm using this in regression tests, I didn't want to set log level to debug4, not to be too verbose.

What do you think ? Should I make it debug4 and update test outputs ?

Reply:
Ok, let's leave this as DEBUG1.